project-base/storefront/components/Pages/Order/TransportAndPayment/helpers.ts

```diff
@@ -317,12 +317,12 @@ export const useLoadTransportAndPaymentFromLastOrder = (
     };
 
     useEffect(() => {
-        if (!isUserLoggedIn) {
+        if (!isUserLoggedIn || !cart) {
             return;
         }
 
         loadTransportAndPaymentFromLastOrder();
-    }, []);
+    }, [!cart]);
```

Comment:
Yeah, because cart changes with every cart change, whereas !cart changes only when cart boolean value changes. We may find a more elegant way. Do you know any?